stix_shifter_modules/darktrace/configuration/config.json

```diff
@@ -0,0 +1,36 @@
+{
+    "connection": {
+        "type": {
+            "displayName": "Darktrace"
+        },
+        "host": {
+            "type": "text",
+            "regex": "^(([a-zA-Z0-9]|[a-zA-Z0-9][a-zA-Z0-9\\-]*[a-zA-Z0-9])\\.)*([A-Za-z0-9]|[A-Za-z0-9][A-Za-z0-9\\-]*[A-Za-z0-9])$"
+        },
+        "help": {
+            "type": "link",
+            "default": "data-sources.html"
+        },
+        "options": {
+            "type": "fields",
+            "result_limit": {
+                "default": 10000,
+                "min": 1,
+                "max": 10000,
+                "type": "number",
+                "previous": "connection.resultSizeLimit"
+            }
```

Comment:
You can just set the value which you are overwriting -
         
            ```
"result_limit": {
                 "max": 10000
             }
```

Reply:
Updated json as suggested.